Mathlib/Algebra/CharP/NeTwo.lean

```diff
@@ -0,0 +1,24 @@
+/-
+Copyright (c) 2025 Alain Chavarri Villarello. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Alain Chavarri Villarello
+-/
+import Mathlib.Algebra.CharP.Two
```

Comment:
Why not adding everything to `Mathlib.Algebra.CharP.Two`? It's just one lemma.

Reply:
This one is on me: I thought it was strange to have a result about characteristic different than two in a file about characteristic two. But it can be moved back there if you think it's okay